### PR TITLE
Add _dendrite types to population

### PR DESCRIPTION
**Pull request by arnaudon**
For compatiblity with NeuritType names of NeuroM/MorphIO

### CODE REVIEW

File: tmd/Neuron/Neuron.py [hunk 1]
```diff
@@ -37,6 +37,16 @@ def dendrites(self):
         '''Get dendrites'''
         return self.apical + self.basal
 
+    @property
+    def apical_dendrite(self):
+        '''Get apical dendrites'''
+        return self.apical
+
+    @property
+    def nasal_dendrite(self):
```
Comment (eleftherioszisis) [inline tmd/Neuron/Neuron.py:46]: https://www.google.com/search?q=nasal&client=ubuntu&hs=HL3&channel=fs&sxsrf=ALiCzsaBfwhz0U4LViIg-40L103RVEsvSg:1660905306538&source=lnms&tbm=isch&sa=X&ved=2ahUKEwicvbS22tL5AhX757sIHS_OC6sQ_AUoAXoECAEQAw&biw=1868&bih=939&dpr=1
:D :D :D

**Review by adrien-berchet — COMMENTED** [review]
LGTM but how do we deal we the `basal` and `apical` properties? Maybe we should remove them or raise a DeprecationWarning and plan to remove them later?

**Comment by arnaudon** [comment]
it's ok to have duplicates, no?

**Comment by eleftherioszisis** [comment]
> LGTM but how do we deal we the `basal` and `apical` properties? Maybe we should remove them or raise a DeprecationWarning and plan to remove them later?

+1 for adding a deprecation warning

**Comment by arnaudon** [comment]
Sure, i can add that, what's the best way to do that? I never wrote a deprecation warning. Thanks!

**Comment by adrien-berchet** [comment]
You can raise a warning with `warnings.warn("The 'basal' property is deprecated, please use 'basal_dendrite' instead", DeprecationWarning)`.

I would try to move the functional code in the `basal_dendrite` property and add the warning in the `basal` property. I never tried to add a warning in a property but I guess it works.

**Comment by arnaudon** [comment]
Ok, now it's a b it more breaking change than before, but as you prefer, I can revert the last commit!

**Review by adrien-berchet — APPROVED** [review]
LGTM
I am just wondering if we should specify in which future version (v3?) these properties will be removed in the deprecation messages?

**Comment by lidakanari** [comment]
LGTM as well. 

 Thanks @arnaudon! @adrien-berchet Do we need to specify when they will be removed? I guess that's the next major version release?

**Comment by adrien-berchet** [comment]
> Do we need to specify when they will be removed? I guess that's the next major version release?

Yeah if we don't specify any version it's implicitly the next major version, so I guess we can keep it implicit.

**Pull request merged**